### PR TITLE
[DOCFIX] [ALLUXIO-1984] Modify cn-doc in docs/cn/Running-Hadoop-Mapreduce-on-Alluxio

### PR DESCRIPTION
**Pull request by 00charis**
[This jira ticket is for Nanjing Hackathon.](https://alluxio.atlassian.net/browse/ALLUXIO-1984?filter=-4)
In the section of Running Hadoop wordcount with Alluxio Locally in docs/en/Running-Hadoop-MapReduce-on-Alluxio.md, there's a statement saying "depends on the hadoop version, you might need to replace ./bin with ./sbin". Update it in the corresponding page of the Chinese version.

### CODE REVIEW

**Comment by ifcharming** [comment]
Thanks @00charis for the contribution! Overall looks good to me.

Can you please add the JIRA ticket link in the PR description? 
Also the format `[DOCFIX] [ALLUXIO-1984] Modify ...` is preferred, per Alluxio convention.

**Comment by 00charis** [comment]
already revised  ：）） @ifcharming

**Comment by ifcharming** [comment]
ok to test

**Comment by ifcharming** [comment]
LGTM, thanks for the improvement!

File: docs/cn/Running-Hadoop-MapReduce-on-Alluxio.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ priority: 1
 
 {% include Running-Hadoop-MapReduce-on-Alluxio/compile-Alluxio-Hadoop-test.md %}
 
-为了方便，我们假设是伪分布式的集群，通过运行如下命令启动：
+为了方便，我们假设是伪分布式的集群，通过运行如下命令启动(根据hadoop的版本，你可能需要把‘./bin’换成‘./sbin’)：
```
Comment (apc999) [inline docs/cn/Running-Hadoop-MapReduce-on-Alluxio.md]: <pre>`./bin`</pre> and <pre>`./sbin`</pre>

**Comment by apc999** [comment]
@00charis thanks for the patch, it seems the quote symbol should be the ASCII quote `. Otherwise it looks good to me

**Comment by ifcharming** [comment]
LGTM, thanks @00charis for the improvement!

**Pull request merged**